### ISSUE TITLE
ImportError: ... pyexpat.so: undefined symbol: XML_SetHashSalt

### ISSUE THREAD

**Issue by cbm755** [cbm755/octsympy#63]
On new regexp branch, I get this error using Matlab 2014a.

Seems its:
https://bugzilla.redhat.com/show_bug.cgi?id=821337

Workaround:

```
octsympy_config python 'LD_LIBRARY_PATH="" python'
```

(to clear the dynamic linking path)

FIXME: test on Mac and Windows

**Comment by cbm755** [comment]
Also see this on Ubuntu 14.04.1 with Matlab 2013a.

**Comment by cbm755** [comment]
Also on Ubuntu 14.04.3 LTS with Matlab 2014b

**Comment by mtmiller** [comment]
Does `setenv PYTHON 'LD_LIBRARY_PATH="" python'` work with the current release in Matlab?

It doesn't work for me in Octave.

Using `setenv LD_LIBRARY_PATH ''` is roughly equivalent, but may cause other things to break unexpectedly.

**Comment by cbm755** [comment]
Yes I think that works.  It also works in Octave using the `sympref ipc system` (which is the only ipc supported on matlab anyway).

I guess this is about the way spaces are treated in `popen2` versus `system`, at least I recall there is some subtly there.

**Comment by mtmiller** [comment]
> works in Octave using the `sympref ipc system`

Thank you, hadn't tried that!
